src/test/pybind/test_rados.py

```diff
@@ -312,8 +312,12 @@ def test_cmpext(self):
     def test_list_objects_empty(self):
         eq(list(self.ioctx.list_objects()), [])
 
-    def test_list_objects(self):
```

Comment:
> @Matan-B: is it fine to take out the `DNM`?

`test_list_objects` will not pass with this patch since it was accidentally changed here.

Please see commit https://github.com/ceph/ceph/pull/57616/commits/8b7694ad799c0cec6be6865c41b1c535d7eb4ea2 which was pushed after the parent PR (of this backport) was merged in main.

Reply:
Same goes for the R backport https://github.com/ceph/ceph/pull/57586